### ISSUE TITLE
How do I install and use this on Windows OS?

### ISSUE THREAD

**Issue by Hookeo** [exercism/exercism.io#2235]
Hey guys,

So I followed the instillation guide.  The only way I can get any response regarding it is to cd to where I saved it (C:\exercism\exercism.exe)

When I type configure, nothing happens.   I'm not sure what I need to do now to actually interact with the program itself.

Thank you for any help :)

**Comment by jwood803** [comment]
@Hookeo were you ok on this issue or still have some questions?

**Comment by Hookeo** [comment]
I figured this out. Now im having issues fetching problems

**Comment by kytrinyx** [comment]
If you paste the output of `exercism debug` we might be able to help you with this. Remember to mask your API key.

**Comment by Hookeo** [comment]
Here is the debug info.

*Ryan Berkley*
*berkleyr@goldmail.etsu.edu <berkleyr@goldmail.etsu.edu>*
*423-598-9930*

**Comment by kytrinyx** [comment]
I'm not seeing it, would you try again?

**Comment by Hookeo** [comment]
I don't know what I did, but I fixed it!

So I was able to get the java exercise.  Within the folder, I have the src,
gradle and readme.  Inside the src there is test and main.  Is my purpose
to edit the EtlTest document to answer the question I've been given?  Now
I'm just trying to figure out what to do with this stuff now that I have it.

*Ryan Berkley*
*berkleyr@goldmail.etsu.edu <berkleyr@goldmail.etsu.edu>*
*423-598-9930*
